### PR TITLE
[smoke-tests] Move local forge tests to local swarm tests

### PR DESCRIPTION
**Pull request by igor-aptos**
### Description

For simpler/easier to use setup

### Test Plan
run tests

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2760)

### CODE REVIEW

File: testsuite/forge/src/backend/local/swarm.rs [hunk 1]
```diff
@@ -106,7 +107,7 @@ impl LocalSwarm {
     where
         R: ::rand::RngCore + ::rand::CryptoRng,
     {
-        println!("Building a new swarm");
+        info!("Building a new swarm");
```
Comment (JoshLind) [inline testsuite/forge/src/backend/local/swarm.rs:110]: Do we know how this affects the `forge-cli` in local mode? i.e., is a logger already created so that these are visible? They're required for manual verification/testing, see: https://github.com/aptos-labs/aptos-core/tree/audit/testsuite/forge-cli/src

**Comment by wrwg** [comment]
Great to see this!

File: testsuite/forge/src/backend/local/swarm.rs (hunk 1 above)
Comment (igor-aptos) [reply testsuite/forge/src/backend/local/swarm.rs:110]: we do ::aptos_logger::Logger::new().init(); in the first line of build() function, so it should be there

**Review by sitalkedia — APPROVED** [review]
> Reviewed 20 of 20 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2760)* status: all files reviewed, 1 unresolved discussion (waiting on @JoshLind, @perryjrandall, @rustielin, and @wrwg)

**Review by wrwg — APPROVED** [review]
The smoketest/src/aptos tests look good to me.

**Comment by davidiw** [comment]
The original intent was for local forge to deprecate local swarm. That would enable us to write tests once and run them in multiple contexts. I don't see a pros / cons here. Could we elaborate on this?

**Comment by igor-aptos** [comment]
worst part of remote forge is:
* how it is started (and how it isn't a test).
* how you can get a non-clean instance
* how configuration (like number of nodes) is separate from the test
The rest of the code should be just fine. if test is never planned to be run on forge, overhead of the above is unnecessary
If some test needs to be run on both local and forge, I would have a single function that accepts Swarm (not Context), and call it both from regular local test, as well as have a wrapper class for forge. That is what I plan to do for tests that I want to run on both.

Basically this makes it much easier to write tests that will not run on forge, but have the similar complexity if you want to run on both.

Ideal way to run remote forge for me would be:

```
#[forge::test(nightly)]
async fn test_a() {
}

#[forge::test(push_blocking)]
async fn test_b() {
}
```


and it has equivalent meaning to
#[ignore]
#[tokio::test]
locally, and if you start it locally, it runs from there.
and nightly and push_blocking runs run those in their k8s environment
